core/mm2Property.js

```diff
@@ -378,7 +378,16 @@ GLOBAL.Property = {
         e.attrs({size: this.displayWidth});
 
         return e;
-      }
+      },
+      adapt: function(_, nu) {
+        if (typeof nu === 'string') {
```

Comment:
I prefer:

return cond ? value1 : value2 ;

When both branches of an if return.